### ISSUE TITLE
What is the unit of time in ExportPulse?

### ISSUE THREAD

**Issue by thnam** [alcap-org/AlcapDAQ#139]
I checked a few pulses from SiR2-S using the ExportPulse module and confused by the unit on the X-axis. 
Is it in nano sec or pico sec? If so the pulse was  60 ms or 60 us?
![screen shot 2014-07-03 at 2 20 15 pm](https://cloud.githubusercontent.com/assets/4444606/3465831/bfb7d2f2-0271-11e4-8007-67d616db2268.png)
From the ODB (run 3200), the frequency was 1.7E+7 -> 60 ns/sample, then the pulse width would be about 60x60 = 3600 ns, much more reasonable to me.

**Comment by benkrikler** [comment]
I think you've spotted a bug in the module.  The function in question is:
[`ExportPulse::PlotTPI`](http://alcap-org.github.io/doxygen/AlcapDAQ/develop/classExportPulse.html#aa9ed6c7e6161f569bd843cf8ce426237).  

You can see we set the x-axis limits to:

```
double max= ((pulse->GetTimeStamp() + num_samples) * fClockTick) - fTimeShift;
double min= -fTimeShift;
```

So we're trying to put this into real global time, but don't quite get it right because we don't include `pulse->GetTimeStamp()` in the axis minimum, which ought to be:

```
double min= (pulse->GetTimeStamp() * fClockTick) - fTimeShift;
```

That would scale everything correctly, but it would still be slightly wrong if the time-stamp of a pulse is not the first sample in the waveform which I think was the case for the FADC's with presampling, right?

**Comment by thnam** [comment]
Thanks Ben!
I also found those offending lines. I was so worried that we have got the wrong clock ticks everywhere, but fortunately that was not the case.

I believe that the time stamp in the TPI is always the time stamp of the first sample, actually that is the definition of a TPI:
https://github.com/alcap-org/AlcapDAQ/blob/develop/analyzer/src/common/TPulseIsland.h#L23-L30

As for the FADCs, there are sorting and stitching to align the samples before creating a TPI
https://github.com/alcap-org/AlcapDAQ/blob/develop/analyzer/src/FADC/TOctalFADCBankReader.cpp#L121
